cc/transport/grpc_streaming_transport.h

```diff
@@ -25,10 +29,41 @@
 
 namespace oak::transport {
 
+template <class Service>
 class GrpcStreamingTransport : public EvidenceProvider, public Transport {
  public:
-  absl::StatusOr<::oak::session::v1::AttestationBundle> GetEvidence() override;
-  absl::StatusOr<std::string> Invoke(absl::string_view request_bytes) override;
+  explicit GrpcStreamingTransport(std::shared_ptr<grpc::Channel> channel)
+      : channel_stub_(Service::NewStub(channel)) {}
```

Comment:
How is this going to work with a different service that uses the same protos but has a different method name? Or will it not yet?

Reply:
By a different method name you mean `Stream`?
https://github.com/project-oak/oak/blob/0464c56c6ac071d153a95957f7ad02890eddf7f8/oak_remote_attestation/proto/v1/service_streaming.proto#L55

Not sure it's possible to do this without requiring to have a method called `Stream`